plugins/signals-backend/README.md

```diff
@@ -0,0 +1,26 @@
+# signals
+
+Welcome to the signals backend plugin!
+
+Signals plugin allows backend plugins to publish messages to frontend plugins.
+
+## Getting started
+
+First install the `@backstage/plugin-signals-node` plugin to get the `SignalService` set up.
+
+Next, add Signals router to your backend in `packages/backend/src/plugins/signals.ts`:
+
+```ts
+import { Router } from 'express';
+import { createRouter } from '@backstage/plugin-signals-backend';
+import { PluginEnvironment } from '../types';
+
+export default async function createPlugin(
+  env: PluginEnvironment,
+): Promise<Router> {
+  return await createRouter({
+    logger: env.logger,
+    service: env.signalsService,
+  });
+}
+```
```

Comment:
Looks like we are missing the steps for updating the `index.ts` for the classic backend setup

Reply:
True, will add!

Reply:
Thanks!